modules/tables/table_11_Utkarsh.html

```diff
@@ -0,0 +1,75 @@
+<style>
+    *{
```

Comment:
using a selector of "*" would change the style for some content that are outside your module.